model-test/src/test/java/hu/blackbelt/judo/meta/jsl/runtime/ActorTests.xtend

```diff
@@ -0,0 +1,213 @@
+package hu.blackbelt.judo.meta.jsl.runtime
+
+import org.eclipse.xtext.testing.InjectWith
+import org.eclipse.xtext.testing.extensions.InjectionExtension
+import org.junit.jupiter.api.^extension.ExtendWith
+import org.eclipse.xtext.testing.util.ParseHelper
+import org.eclipse.xtext.testing.validation.ValidationTestHelper
+import com.google.inject.Inject
+import hu.blackbelt.judo.meta.jsl.jsldsl.ModelDeclaration
+import org.junit.jupiter.api.Test
+import hu.blackbelt.judo.meta.jsl.jsldsl.JsldslPackage
+import hu.blackbelt.judo.meta.jsl.validation.JslDslValidator
+
+@ExtendWith(InjectionExtension)
+@InjectWith(JslDslInjectorProvider)
+class ActorTests {
+    @Inject extension ParseHelper<ModelDeclaration>
+    @Inject extension ValidationTestHelper
+	
+	@Test
```

Comment:
Please remove tabs to spaces.
Here is a tutorial, how can set the Eclipse: https://readlearncode.com/code-and-stuff/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs/